_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,390 @@
+---
+layout: default
+title: Creating reports with the Reporting CLI
+nav_order: 75
+---
+
+# Creating reports with the Reporting CLI
+
+You can programmatically create dashboard reports in PDF or PNG format with the Reporting CLI without using OpenSearch Dashboards or the Reporting plugin. This allows you to create reports automatically within your email workflows.
+
+If you want to download a CSV file, you need to have the Reporting plugin installed.
+{: .note }
+
+For any dashboard view, you can request a report in PNG or PDF format to be sent to an email address. This can be useful for sending reports to multiple email recipients with an email alias. The only dashboard view that supports creating a CSV report is the **Discover** view.
+
+With the Reporting CLI, you can specify options for your report in the command line. The report is sent to an email address as a PDF attachment by default. You can also request a PNG image or a CSV file with the `--formats` argument or embed the image in the email body instead of attaching it.
```

Comment:
can we remove `or embed the image in the email body instead of attaching it.` ? After adding email template, report will be embedded in email for quick view and also as attachment.